Source/WebCore/platform/graphics/cocoa/IOSurface.mm

```diff
@@ -469,13 +469,7 @@ static IntSize computeMaximumSurfaceSize()
 
 IOSurfaceID IOSurface::surfaceID() const
 {
-// FIXME: Should be able to do this even without the Apple internal SDK.
-// FIXME: Should be able to do this on watchOS and tvOS.
-#if PLATFORM(IOS_FAMILY) && (!USE(APPLE_INTERNAL_SDK) || PLATFORM(WATCHOS) || PLATFORM(APPLETV))
-    return 0;
-#else
```

Comment:
Is there cleanup to be done in the SPI header?

Reply:
There's an IOSurfaceID typedef for watch and tv in `WTF/wtf/spi/cocoa/IOSurfaceSPI.h`. Can probably remove that as well.